<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view</class>
 <widget class="QWidget" name="view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>280</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_cpu">
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <attribute name="title">
     <string>CPU</string>
    </attribute>
    <widget class="QLabel" name="header_tab_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CPU info</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>93</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU usage</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="cpu_usage_custom" native="true">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>91</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>93</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU temp</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="cpu_temperature_custom" native="true">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>91</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>172</y>
       <width>82</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="cpu_freq_label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU freq:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cpu_load_label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU load:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>172</y>
       <width>62</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="cpu_freq_label_value">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>10 Ghz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cpu_load_label_value">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2.2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_mem">
    <attribute name="title">
     <string>MEM</string>
    </attribute>
    <widget class="QLabel" name="header_tab_label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MEM info</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>93</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>RAM </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="ram_custom" native="true">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>91</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>58</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="header_tab_label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Used: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="header_tab_label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cached:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="header_tab_label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Swap:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="header_tab_label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Free:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>83</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="ram_used_layout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ram_used">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1GB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_used_slash">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_used_total">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>4GB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ram_cached_layout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ram_cached">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1GB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_cached_slash">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_used_total_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>4GB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ram_swap_layout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ram_swap">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1GB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_swap_slash">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_used_total_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>4GB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ram_free_layout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ram_free">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1GB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_free_slash">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ram_used_total_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>4GB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_sys">
    <attribute name="title">
     <string>SYS</string>
    </attribute>
    <widget class="QLabel" name="header_tab_label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>System info</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>109</width>
       <height>87</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="kernel_version_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kernel version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uptime_label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Uptime:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="load_average_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Load average:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>91</width>
       <height>87</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="kernel_version_label_value">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>11.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uptime_label_value">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>11.33</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="load_average_label_value">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>4.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="uptime_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update time: 09:10:50</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
